examples/generate.sh

```diff
@@ -121,19 +127,19 @@ curl --fail -Ss -L -o "${COMPONENTS_CERT_MANAGER_GENERATED_FILE}" https://github
 echo "Downloaded ${COMPONENTS_CERT_MANAGER_GENERATED_FILE}"
 
 # Generate Cluster API provider components file.
-"$KUSTOMIZE" build "github.com/kubernetes-sigs/cluster-api/config/?ref=release-0.3" | "$ENVSUBST" > "${COMPONENTS_CLUSTER_API_GENERATED_FILE}"
+"$KUSTOMIZE" build "github.com/kubernetes-sigs/cluster-api/config/default/?ref=master" | "$ENVSUBST" > "${COMPONENTS_CLUSTER_API_GENERATED_FILE}"
```

Comment:
Unfortunately no, previously it was pointing to branch (release-0.3) where all config was under `/config` folder, but since the API version uplift CAPI does not have anything like release-0.4 branch as of now and we need to use master to be able to fetch configuration under `config/default`